src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -244,6 +244,8 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
   /** A view representing the elements of this collection. */
   def view: View[A] = View.fromIterator(coll.iterator())
 
+  def transform[B](vt: Iterable[A] => View[B]): CC[B] =  fromIterable(vt(coll))
```

Comment:
Except in this case it starts out as an `Iterable` and ends up as one.  The intermediate representation as `View`s is really an implementation detail.  The closest equivalent in Clojure's transducer kit would be `sequence`, but that's even worse.